config.yml

```diff
@@ -51,9 +71,12 @@ steps:
           branch: '%payload.pull_request.head.ref%'
       - type: closeIssue
         issue: Welcome
-
   - title: Merge your pull request
     description: Merge the pull request that customizes your homepage.
+    translations:
+      es:
+        title: Combina tu pull request
+        description: Combina el pull request que personaliza tu página inicial.
```

Comment:
🍺 ¿Combina? Not sure 🤔

Reply:
That is what I normally use in my classes, and I think it helps students understand what `merge` is actually doing. _Pro Git_ translates to _fusionar_, so that's definitely another option.

Reply:
Totally understand. I do not usually discuss programming topics with others in Spanish so I have no idea, I would have to ask the community what terms people commonly use.